package.json

```
@@ -104,7 +105,7 @@
     "path-browserify": "^1.0.1",
     "pixelmatch": "4.0.2",
     "postcss": "^8.2.8",
-    "prettier": "^2.2.1",
+    "prettier": "2.3.2",
```

I know the docs state that "it’s important to have a locked down version of Prettier", but isn't that the point of `package-lock.json`?

---

Prettier is interesting in that it's possible to have breaking changes in minor versions (like TypeScript and ts-jest), so I've made it a habit to lock it down to a specific version. Maybe a better alternative is:
```suggestion
    "prettier": "~2.3.2",
```
Thoughts?

---

I don't have a strong opinion and I'm fine with either - my thinking was just that `package-lock.json` will lock it down to a specific version.